frr/Dockerfile

```diff
@@ -13,3 +13,5 @@ RUN sed -i -e 's/service/no service/' /etc/frr/vtysh.conf \
 
 RUN apt clean && \
     rm -rf /tmp/* /var/lib/apt/lists/* /var/tmp/*
+
+COPY workaround.conf /etc/systemd/system/frr.service.d/workaround.conf
```

Comment:
Hi @BuonHobo,
can you rename the `workaround.conf` file into `su_frr.conf`?

Thanks,
Mariano.

Reply:
Done!